tests/README.md

```diff
@@ -68,26 +70,24 @@ make run-<unit|regression|integration|end-to-end>
 
 Additionally tests can be filtered via tags using the `-<tags,...>` suffix to the `run-<target>` command.
 
-Run individual tests:
-
-```bash
-make <file/path> # without trailing .bats, for generated files use the .gen ending
-```
-
 Clean up:
 
 ```bash
 # remove dependencies and generated files
 make clean
 
-# remove dependencies
-make clean-dep
-
 # remove generated files
 make clean-gen
 ```
```

Comment:
Can this entire section not be removed now when we commit the generated tests? Or is there still a good reason to clean things up sometimes?